std_msgs/Header header 	                        # Message header including timestamp of detection & img frame
interruptibility_msgs/PersonID person_id		# ID of the person that we are making a prediction on
uint8 prediction		                        # Value of prediction, 0 or 1